README.md

```diff
@@ -1,48 +1,54 @@
-This repo is where the code to Magenta Studio is maintained. For information, support and installation instructions of the tools themselves go to [this website](https://magenta.tensorflow.org/studio/)
+This repo is where the code to Magenta Studio is maintained. For information, support and installation instructions of the tools themselves go to [this website](https://magenta.tensorflow.org/studio/).
 
 This is not a Google product. 
 
 ## Overview and Organization
 
-The Magenta Studio plugins are Electron apps. Electron has the advantage of being a cross platform solution that works well with [magenta.js](https://github.com/tensorflow/magenta-js). The code for each of the plugins is located in a folder with the same name. 
+Magenta Studio is a set of plugins contained in a [Max for Live](https://www.ableton.com/en/live/max-for-live/) Device. They are contained in a single web application that runs in the Max environment via Chromium Embedded Framework (CEF). All of the front-end code is contained in the `client/` folder, which contains the five plugins, and common UI and communication components which can be found in the `client/components/` folder. These objects are built using [Polymer's LitElement](https://github.com/Polymer/lit-element). 
```

Comment:
Let's update the link from "Polymer's LitElement" to just say "[lit](https://lit.dev/)"